plugins/Login/tests/UI/Login_spec.js

```diff
@@ -119,4 +123,64 @@ describe("Login", function () {
             page.load('');
         }, done);
     });
+
+    it("should show brute force log url when there are no entries", function (done) {
+        testEnvironment.testUseMockAuth = 1;
+        delete testEnvironment.queryParamOverride;
+        delete testEnvironment.bruteForceBlockThisIp;
+        delete testEnvironment.bruteForceBlockIps;
+        testEnvironment.save();
+
+        expect.screenshot("bruteforcelog_noentries").to.be.capture(function (page) {
+            page.load(bruteForceLogUrl);
+        }, done);
+    });
+
+    it("should show brute force log url when there are entries", function (done) {
+        testEnvironment.testUseMockAuth = 1;
+        testEnvironment.bruteForceBlockIps = 1;
+        delete testEnvironment.bruteForceBlockThisIp;
+        delete testEnvironment.queryParamOverride;
+        testEnvironment.save();
+
+        expect.screenshot("bruteforcelog_withentries").to.be.capture(function (page) {
+            page.load(bruteForceLogUrl);
+        }, done);
+    });
+
+    it("should show error when trying to attempt a log in through API", function (done) {
+        testEnvironment.testUseMockAuth = 1;
+        testEnvironment.bruteForceBlockThisIp = 1;
+        delete testEnvironment.bruteForceBlockIps;
+        delete testEnvironment.queryParamOverride;
+        testEnvironment.save();
+
+        expect.screenshot("bruteforcelog_blockedapi").to.be.capture(function (page) {
+            page.load(apiAuthUrl);
+        }, done);
+    });
```

Comment:
I think this one could be tested in a system test w/ `Http::sendRequest()`, in case you don't want to keep the screenshot around.